image_services/opera/README.md

```diff
@@ -0,0 +1,54 @@
+OPERA GIS Services
+=========
+**_A workflow for programatically standing up an OPERA Image Service_**
+-----
+
+## Steps for creating a service
+Below are the steps to create an image service using OPERA data. 
+
+### 1. Check permissions and server set up
+Prior to running the workflow, ensure that permissions are granted to pull from the EDC-hosted S3 bucket `asf-cumulus-prod-opera-products` and push and pull from a separate S3 bucket for overviews. 
+
+Ensure that the `arcpy` conda environment is created and activated. 
+```
+cd /home/arcgis/gis-services/image_server/
+mamba env create -f environment.yml
+conda env activate arcpy
+```
+
+Check there is a server connection json file. This workflow will assume that this file will be in the home directory (`/home/arcgis`). If there is another location for this file, make sure to enter its location as an argument [when running the code](#Run-the-image-services-code).
```

Comment:
"server connection json file" is a concept we made up, I think it's worth saying something like "this file contains the connection information needed to publish the final mosaic dataset to ArcGIS Image Server":

```
{
    "url": "https://localhost:6443/arcgis/admin",
    "username": "...",
    "password": "..."
}
```